File: Back/src/ProEvents.Domain/Event.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ProEvents.Domain
{
    //[Table("Events")]
    public class Event
    {
        //[Key]
        public int Id { get; set; }    
        public string Local {get; set;}
        public DateTime? Date { get; set; }

        //[NotMapped]
        //public int DaysCount {get; set;} 

        [Required]
        [MaxLength(50)]
        public string Subject { get; set; }
        public int AmountOfPeople { get; set; }
        public string ImageUrl { get; set; }
        public string Phone { get; set; }   
        public string Email { get; set; }
        public IEnumerable<Lot> Lots { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<SpeakerEvent> SpeakerEvents { get; set; }
    }
}

File: Front/ProEvents-App/src/app/models/Event.ts
import { Lot } from "./Lot";
import { SocialNetwork } from "./SocialNetwork";
import { Speaker } from "./Speaker";

export interface Event {
  id: number;
  local: string;
  date?: Date;
  subject: string;
  amountOfPeople: number;
  imageUrl: string;
  phone: string;
  email: string;
  lots: Lot[];
  socialNetworks: SocialNetwork[];
  speakerEvents: Speaker[];
}
